examples/asynccopy/asynccopy.cpp

```diff
@@ -1,18 +1,42 @@
+/* To the extent possible under law, Alexander Matthes has waived all
+ * copyright and related or neighboring rights to this example of LLAMA using
+ * the CC0 license, see https://creativecommons.org/publicdomain/zero/1.0 .
+ *
+ * This example is meant to be "stolen" from to learn how to use LLAMA, which
+ * itself is not under the public domain but LGPL3+.
+ */
+
+/** \file asynccopy.cpp
+ *  \brief Asynchronous bluring example for LLAMA using ALPAKA.
+ */
+
 #include <iostream>
 #include <utility>
 #include <list>
 
+/// defines whether cuda shall be used or openmp
 #define ASYNCCOPY_CUDA 0
```

Comment:
I usually prefer to put such controls in a
```C++
#ifndef VAR
#define VAR default
#endif 
```

block so one can control them without warnings externally.

Reply:
You are right, I added those so I can quick change them in file while debugging / benchmarking, but I should go your way. Would you like to open an issue for this? Then I will tackle this soon. :wink:

Reply:
My pleasure, done in #16